tools/dev/tests/test_poetrypkg.py

```diff
@@ -67,6 +68,7 @@ def _assert_pkg_ok(
 
 @pytest.mark.skipif(sys.version_info <= (3, 8), reason="Developer Python minimum is 3.8")
 @pytest.mark.skipif(sys.platform == 'win32', reason="No Windows support")
+@pytest.mark.skipif(os.environ.get('CI', '') == 'true', reason="Skip in CI")
```

Comment:
pipx install of poetry doesn't work for the auto discovery on poetry, and poetry-core is not sufficient.